### PR TITLE
Add global flag

### PR DESCRIPTION
**Pull request by teresaromero**
Resolves https://github.com/okteto/okteto/issues/2182
Enable builds with manifest v2 and support the --global tag

### CODE REVIEW

**Comment by teresaromero** [comment]
Can't be merged until `--global` is available at cli okteto build

**Comment by teresaromero** [comment]
> @teresaromero can we merge this one now cli 2.0 is available?

this PR only adds the flag of `--global` to the argument list, but we should check that there is compatibility between actions v1 and v2, as default values are different meaning in 2.0. `path` and `file` at v2 are pointing to services and manifest file, instead of Dockerfile and it's path...

**Comment by pchico83** [comment]
@teresaromero I would update the action to follow the cli 2.0 syntax. people can ping to a previous action version if needed

File: action.yml [hunk 1]
```diff
@@ -15,6 +15,10 @@ inputs:
   buildargs:
     description: 'Use buildargs when you want to pass a list of environment variables as build-args'
     required: false
+  global:
+    description: 'Use global when you want to push the image to the global registry'
```
[the remark this answers is not in the document]
Comment (teresaromero) [reply action.yml]: updated this, also added the arg to readme which was missing.

cc: @pchico83 @jLopezbarb pls review this changes

**Review by pchico83 — APPROVED** [review]

File: entrypoint.sh [hunk 2]
```diff
@@ -25,5 +27,12 @@ if [ -z "$tag" ]; then
   params=$(eval echo --progress plain -f "$file" "$BUILDPARAMS" "$path")
 fi
 
+if [ "$global" = "true" ]; then
```
Comment (jLopezbarb) [inline entrypoint.sh:29]: I don't know if it's supported on sh to use = I think you have to use -eq operator

Comment (jLopezbarb) [inline entrypoint.sh:29]: Should we support other true values like "1", "t", "T", "true", "TRUE", "True"

**Review by jLopezbarb — APPROVED** [review]

File: entrypoint.sh (hunk 2 above)
Comment (jpf-okteto) [reply entrypoint.sh:29]: `=` is ok for strings.

From https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html

> string1 = string2
> True if the strings are equal. ...
> 
> arg1 OP arg2
> OP is one of ‘-eq’, ‘-ne’, ‘-lt’, ‘-le’, ‘-gt’, or ‘-ge’. ... Arg1 and arg2 may be positive or negative integers. ...

File: README.md [hunk 3]
```diff
@@ -35,6 +35,9 @@ A list of environment variables as build-args
         buildargs: PACKAGE_NAME,VCS_REF
 ```
 
+### `global`
+
+When true will make the image available to everyone in your team. Default `false`.
```
Comment (rberrelleza) [inline README.md:40]: Can you mention that you need to be an admin user to use this?

Comment (pchico83) [reply README.md:40]: @teresaromero could you take care of ^^?

**Pull request merged**